fluent-plugin-kubernetes_metadata_filter.gemspec

```diff
@@ -5,7 +5,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 Gem::Specification.new do |gem|
   gem.name          = 'fluent-plugin-kubernetes_metadata_filter'
-  gem.version       = '2.7.0'
+  gem.version       = '2.7.1'
```

Comment:
Please revert this change otherwise lgtm

Reply:
Oops, good point. This is backwards compatible so no need to push gem version forward.